src/modules/data.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable */
+const tasks = [
+  {
+    description: 'Buy milk',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Do laundry',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Learn webpack',
+    completed: false,
+    index: 2,
+  },
+];
+
+function renderTasks() {
+  const todoList = document.getElementById('todo-list');
```

Comment:
- [ ] You did a great job configuring Webpack in this project. :+1: According to the project requirements, the array of tasks created here should be displayed on page load. Currently, they are not getting displayed when the page is loaded. I don't think there is an HTML element with the ID `todo-list`. Please address this issue. 🤝️

**Requirements**
![Screenshot from 2023-05-11 07-34-15](https://github.com/MoyasiGinko/ToDoList-Project/assets/17243325/22268de4-1a60-4bcb-b0d6-d4ceaba1e8b3)